src/NearestNeighbors.jl

```diff
@@ -161,7 +175,7 @@ metadata_pkg.((KNNRegressor, KNNClassifier),
 
 metadata_model(KNNRegressor,
     input   = Table(Continuous),
-    target  = Union{AbstractVector{Continuous}, AbstractVector{<:AbstractArray{Continuous}}},
+    target  = Union{AbstractVector{Continuous}, Table{Continuous}},
     weights = true,
```

Comment:
`Table(Continuous)` instead of `Table{Continuous}` https://github.com/alan-turing-institute/ScientificTypes.jl#more-on-the-table-type